app/models/submission_entry.rb

```diff
@@ -1,17 +1,20 @@
 class SubmissionEntry < ApplicationRecord
   include AASM
 
+  TYPES = %w[invoice order other].freeze
+
   belongs_to :submission
   belongs_to :submission_file, optional: true
   belongs_to :customer, primary_key: :urn, foreign_key: :customer_urn, required: false, inverse_of: :submission_entries
   has_one :framework, through: :submission
 
   validates :data, presence: true
-  validates :entry_type, inclusion: { in: %w[invoice order] }, allow_blank: true
+  validates :entry_type, inclusion: { in: TYPES }, allow_blank: true
 
   scope :sheet, ->(sheet_name) { where("source->>'sheet' = ?", sheet_name) }
   scope :invoices, -> { where(entry_type: 'invoice') }
   scope :orders, -> { where(entry_type: 'order') }
+  scope :others, -> { where(entry_type: 'other') }
```

Comment:
```ruby
TYPES.each do |type|
  scope type.pluralize.to_sym, -> { where(entry_type: type) }
end
```

... is this a terrible idea..?

Reply:
It's still three lines and it's less obvious. When we look at it now we can see what the three scopes are in exact language. Maybe it'd pay for itself at 4 `TYPES`, or 10?

Reply:
Anyway it's a 2/10 on the Watkins scale